tests/state/test_get_states.py

```diff
@@ -0,0 +1,372 @@
+"""
+Provide tests for command line interface's get a list of states command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    DEV_BRANCH_NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+ADDRESS_WITH_STATE = '0000000000000000000000000000000000000000000000000000000000000000000001'
+
+
+def test_get_states():
+    """
+    Case: get a list of states.
+    Expect: list of states is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--node-url',
+        DEV_BRANCH_NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    result_state_addresses = json.loads(result.output).get('result')
+    result_state_addresses = result_state_addresses[0]
```

Comment:
`result_state_addresses` > `first_state_address`. You should mention it is first as said in `case` that you start from some state.

Reply:
@dmytrostriletskyi, done.